recipes/datreant.core/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.7.0" %}
+{% set sha256 = "425fa5b8e64c06eb4c7c07c2ec5209281428c7c63ce6190b6486bb096f1bebfa"%}
+
+package:
+  name: datreant.core
+  version: {{ version }}
+
+source:
+  fn: datreant.core-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/datreant.core/datreant.core-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - asciitree
+    - pathlib2
+    - scandir
+    - six
+    - fuzzywuzzy
+    - pytest
+    - datreant
+
+  run:
+    - python
+    - asciitree
+    - pathlib2
+    - scandir
+    - six
+    - fuzzywuzzy
+    - python-levenshtein
+    - datreant
+    - pytest
+
+test:
+  imports:
+    - datreant.core
+    - datreant.core.backends
+    - datreant.core.tests
+
+about:
+  home: http://datreant.org/
+  license: BSD 3-Clause
```

Comment:
It ensures that the license file shows up in `sdist`s and other packages generated from the source. This is important as it is required by the BSD license. Further it will make it easy for this package to comply with the license as we can do something like `license_file: LICENSE` to include it here.